diff --git a/python/repositories.bzl b/python/repositories.bzl
index d4b113f..e451c84 100644
--- a/python/repositories.bzl
+++ b/python/repositories.bzl
@@ -97,7 +97,7 @@ def _python_repository_impl(rctx):
 
     # Make the Python installation read-only.
     if "windows" not in rctx.os.name:
-        exec_result = rctx.execute(["chmod", "-R", "ugo-w", "lib"])
+        exec_result = rctx.execute(["/bin/chmod", "-R", "ugo-w", "lib"])
         if exec_result.return_code:
             fail(exec_result.stderr)
 
